Ginger/Ginger/DragDropLib/DragDrop2.cs

```diff
@@ -29,14 +30,14 @@ public static class DragDrop2
     {
         private static Point _startPoint;
         private static bool IsDragging;
-        public static DragInfo DragInfo;
+        public static DragInfo mDragInfo;
```

Comment:
why "m" was added to "DragInfo" if it is a Public field which been accessed from outside? we add m only to private fields.